ansible/playbooks/roles/preflight/tasks/main.yml

```diff
@@ -30,6 +30,25 @@
 
 - include_tasks: check-os.yml
 
+- include_vars:
+    file: roles/common/vars/main.yml
+    name: common_vars
+
+- name: Validate network configuration
+  when: common_vars.provider == "any"
```

Comment:
So far we used `any` also for clusters created with Epiphany infrastructure modules. If they are still supported by epicli and public IPs are used then the assertion below may fail.